manuscript/draft.Rmd

```diff
@@ -259,59 +260,63 @@ on these time scales once observer differences are accounted for.
 When comparing forecasts for richness across sites all methods performed well
 (Figure 2; R^2 XX-XX%). However stacked and joint SDMs and the macroecological
 model both failed to successfully forecast the highest richness sites resulting
-in a noteable clustering of predicted values near ~60 species and the poorest
+in a notable clustering of predicted values near ~60 species and the poorest
 performance in the cross-site comparison (R^2 XX-XX%).
 
 For most combinations of metric and forecast time lag, the best-performing model
-for forecasting within site variability was either the "average" baseline model
-or the ARIMA model (Figure 3). In general, each model's error increased
-more-or-less linearly with time, although the slopes and initial error rates
-varied across models (Figure 3). The site-level models (i.e. the baselines and
-the ARIMA methods) tended to do very well (Figure 3) on short time horizons,
-especially in terms of RMSE.
+for forecasting within-site variability was either the "average" baseline model
+or the ARIMA model [[Not sure what the bit about within-site variability means
+here. Also, didn't Naive usually win when observer effects weren't corrected
+for?]] (Figure 3). In general, each model's error increased more-or-less
```

Comment:
The "within-site variability" part is meant to distinguish this analysis from that in the previous paragraph on among site comparisons.

Yep, the Naive is usually the winner when not correcting for observer effects, but my thinking was that our main results are those when we've corrected for observer effects. We want to discuss those changes in the context of the importance of the observer model, but I don't think we want to mix that discussion in with the other results. That's why I held discussion of the non-observer corrected data out for it's own paragraph. Does that make sense?